src/main/java/exceptions/EmptyTaskException.java

```diff
@@ -0,0 +1,5 @@
+package exceptions;
+
+public class EmptyTaskException extends Exception{
```

Comment:
Formatting issue with the curly brace! I'm also not sure why the exception class is empty.

This applies to all the exception classes.